### PR TITLE
Fixes #1562

### LINKED ISSUE #1562: When private games disabled, they still show up in coop list without indicator.
When "Show private games" is enabled, games show correctly in coop list.
But when disabled, private games in coop list still show up (but without lock symbol so if you cannot distinguish between private and non-private games).

### PR DESCRIPTION
**Pull request by 1-alex98**

### CODE REVIEW

File: src/main/java/com/faforever/client/game/GamesTableController.java [hunk 1]
```diff
@@ -84,10 +84,10 @@ public Node getRoot() {
   }
 
   public void initializeGameTable(ObservableList<Game> games) {
-    initializeGameTable(games, null);
+    initializeGameTable(games, null, true);
   }
 
-  public void initializeGameTable(ObservableList<Game> games, Function<String, String> coopMissionNameProvider) {
+  public void initializeGameTable(ObservableList<Game> games, Function<String, String> coopMissionNameProvider, Boolean listenToFilterPreferences) {
```
Comment (Geosearchef) [inline src/main/java/com/faforever/client/game/GamesTableController.java]: use primitive boolean

**Review by Geosearchef — CHANGES_REQUESTED** [review]

**Review by Brutus5000 — DISMISSED** [review]
Apart from Geosearchefs suggestion, looks good.

**Comment by Katharsas** [comment]
Ok so the coop table does not react to either the private or mod flag anymore right? Looks good.

**Comment by 1-alex98** [comment]
Yes

**Review by Katharsas — DISMISSED** [review]
Ok.

**Pull request merged**